src/mongocxx/test/spec/operation.cpp

```diff
@@ -1275,7 +1275,7 @@ document::value operation_runner::_create_index(const document::view& operation)
     return _coll->create_index(*session, keys, opts.extract());
 }
 
-operation_runner::operation_runner(collection* coll) : _coll(coll) {}
+operation_runner::operation_runner(collection* coll) : operation_runner(nullptr, coll) {}
```

Comment:
A [delegating constructor](https://docs.microsoft.com/en-us/cpp/cpp/delegating-constructors?view=vs-2019) that sets `_db` to `nullptr` fixes the issue.